crowdnotifier-sdk/src/main/java/org/crowdnotifier/android/sdk/model/ProblematicEventInfo.java

```diff
@@ -6,16 +6,16 @@
 	private byte[] secretKeyForIdentity;
 	private long startTimestamp;
 	private long endTimestamp;
-	byte[] encryptedMessage;
+	byte[] encryptedAssociatedData;
 	byte[] nonce;
```

Comment:
If possible can we rename `nonce` to `cipherTextNonce` (is in the iOS version) or `associatedDataNonce` or `encryptedAssociatedDataNonce` to make it a bit more descriptive?

Reply:
Absolutely